index.html

```diff
@@ -0,0 +1,473 @@
+                <hr class="line">
+                <div class="inline-text">
+                    <p>Unlock more free articles.</p>
+                    <p class="link">Create an account or log in</p>
+                </div>
+                <hr class="line">
+                <p>Inflation has been the workhorse of cosmology for 35 years, though many, including Dr. Guth, wondered whether it could ever be proved.</p>
+
+                <p>If corroborated, Dr. Kovac’s work will stand as a landmark in science comparable to the recent discovery of dark energy pushing the universe apart, or of the Big Bang itself. It would open vast realms of time and space and energy to science and speculation.</p>
+
+                <p>Confirming inflation would mean that the universe we see, extending 14 billion light-years in space with its hundreds of billions of galaxies, is only an infinitesimal patch in a larger cosmos whose extent, architecture and fate are unknowable. Moreover, beyond our own universe there might be an endless number of other universes bubbling into frothy eternity, like a pot of pasta water boiling over.</p>
+            </div>
+            <div class="a3">
+                <h3>‘As Big as It Gets’</h3>
+                <p>In our own universe, it would serve as a window into the forces operating at energies forever beyond the reach of particle accelerators on Earth and yield new insights into gravity itself. Dr. Kovac’s ripples would be the first direct observation of gravitational waves, which, according to Einstein’s theory of general relativity, should ruffle space-time.</p>
+
+                <p>Marc Kamionkowski of Johns Hopkins University, an early-universe expert who was not part of the team, said, “This is huge, as big as it gets.”</p>
+
+                <p>He continued, “This is a signal from the very earliest universe, sending a telegram encoded in gravitational waves.”</p>
+
+                <p>The ripples manifested themselves as faint spiral patterns in a bath of microwave radiation that permeates space and preserves a picture of the universe when it was 380,000 years old and as hot as the surface of the sun.</p>
+
+                <p>Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.</p>
+
+                <div id="article-small">
+                    <div class="pos-text">
+                        <h3>The Theory of Inflation</h3>
+                        <span>Astronomers have found evidence to support the theory of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</span>
+                    </div>
+                    <div class="co1"><img alt="inflation1" src="images/1.png"></div>   
```

Comment:
- [ ] **MANDATORY** The purpose of this project is to learn to position the elements, this inflation part should be done using your skills to position the text and images in a separate way (I would suggest you to use a grid). Please, do not use an image of the text. Also, check your positioning:
![image](https://user-images.githubusercontent.com/53256139/87978269-c3ac2c80-ca95-11ea-85e3-7139676b16ea.png)